base/REPLCompletions.jl

```diff
@@ -228,12 +228,19 @@ function completions(string, pos)
         # also search for packages
         s = string[startpos:pos]
         if dotpos <= startpos
-            append!(suggestions, filter(isdir(Pkg.dir()) ? readdir(Pkg.dir()) : Union(ASCIIString,UTF8String)[]) do pname
-                pname[1] != '.' &&
-                pname != "METADATA" &&
-                pname != "REQUIRE" &&
-                startswith(pname, s)
-            end)
+            for dir in [Pkg.dir(), LOAD_PATH, pwd()]
+                !isdir(dir) && continue
+                for pname in readdir(dir)
+                    if pname[1] != '.' && pname != "METADATA" &&
+                          pname != "REQUIRE" && startswith(pname, s)
```

Comment:
Not a big deal, but it doesn't seem the rest of the code base follows this convention. We need a Julia style guide.